build-image-src/Dockerfile-go1x

```diff
@@ -0,0 +1,50 @@
+FROM amazon/aws-sam-cli-emulation-image-go1.x
+
+RUN yum groupinstall -y development && \
+  yum install -d1 -y \
+  tar \
+  gzip \
+  unzip \
+  python3 \
+  jq \
+  git \
+  grep \
+  curl \
+  make \
+  rsync \
+  gcc-c++ \
+  binutils \
+  procps \
+  libgmp3-dev \
+  zlib1g-dev \
+  liblzma-dev \
+  libxslt-devel \
+  libmpc-devel \
+  && yum clean all
```

Comment:
Can you please ensure that all these dependencies are mentioned in our [Attribution file](https://github.com/aws/aws-sam-build-images/blob/develop/build-image-src/ATTRIBUTION.txt). Probably if it is same as other images then it is fine, if there is anything new then we need to add it.

Reply:
Added jq and go packages.